store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/util/MergeElementFunction.java

```diff
@@ -77,16 +91,37 @@ public ApplyViewToElementsFunction(final Map<String, Object> context) throws Gaf
             }
             // Validate the supplied context before using
             validate(context);
+
+            updateViewWithValidationFromSchema(context);
+
             this.context = Collections.unmodifiableMap(context);
         } catch (final Exception e) {
             throw new GafferCheckedException("Unable to create TemporaryResultsGraph", e);
         }
 
     }
 
+    private static void updateViewWithValidationFromSchema(final Map<String, Object> context) {
```

Comment:
The method is not used by other classes so it's private. and the method is not tied to the instance of this class so it can be static.

It only changes the values of a context object. After that object has been modified it is then assigned to a persistent field.

Reply:
please examine https://github.com/gchq/Gaffer/pull/3165